### PR TITLE
Add MP JWT 1.2 standalone blog to staging

### PR DESCRIPTION
**Pull request by ayoho**
For #1168

Draft site: https://draft-openlibertyio.mybluemix.net/blog/2021/03/26/MP-JWT-1.2.html

image
image
image
image
image
image
image
image
image

### CODE REVIEW

File: posts/2021-03-26-MP-JWT-1.2.adoc [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+The `mp.jwt.verify.audiences` property allows you to create list of allowable audience values. At least one of these must be found in the `"aud"` claim of the JWT. Previously, this had to be configured in the server.xml file. Now you can configure the audiences in the MicroProfile Config property as follows:
+[source]
+----
+mp.jwt.verify.audiences=conferenceService, adminService
+----
+
+[#mp-jwt-decrypt-key-location]
+=== mp.jwt.decrypt.key.location
+
+To support JSON Web Encryption tokens, the `mp.jwt.decrypt.key.location` property allows you to specify the location of the Key Management Key. This is the private key that is used to decrypt the Content Encryption Key, which is then used to decrypt the JWE ciphertext. The private key must correspond to the public key that is used to encrypt the Content Encryption Key.
```
Comment (mbroz2) [inline posts/2021-03-26-MP-JWT-1.2.adoc:78]: shouldn't "Key Management Key" be "Private Management Key"

**Review by mbroz2 — CHANGES_REQUESTED** [review]

File: posts/2021-03-26-MP-JWT-1.2.adoc (hunk 1 above)
Comment (teddyjtorres) [reply posts/2021-03-26-MP-JWT-1.2.adoc:78]: "Key Management Key" is a technical term derived from the JWE specification and we want to be specific as to what Key, from the specification, this is referring to.

This is what the MP JWT 1.2 specification refers to in https://download.eclipse.org/microprofile/microprofile-jwt-auth-1.2/microprofile-jwt-auth-spec-1.2.html,

> "Key management key - public RSA key which is used to encrypt a content encryption key. mp.jwt.decrypt.key.location must point to a private RSA key matching this key."

Comment (ayoho) [reply posts/2021-03-26-MP-JWT-1.2.adoc:78]: "Key management key" is the noun phrase we've used throughout our docs. The [JWE spec](https://tools.ietf.org/html/rfc7516#section-2) defines the phrase "key management mode" as the "method of determining the Content Encryption Key value to use". Building off of that, we decided the key used to wrap the Content Encryption Key would then be the key management key. Our `<jwtBuilder>` and `<mpJwt>` elements have corresponding `keyManagementKeyAlias` attributes.

Comment (mbroz2) [reply posts/2021-03-26-MP-JWT-1.2.adoc:78]: ah, thanks for the explanation.  Key Management Key sounded odd to me, like a mistake, and when I searched for jwt.decrypt.key.location, I saw that smallrye seemed to refer to it as the "Private Decryption Key" and in an [MP issue](https://github.com/eclipse/microprofile-jwt-auth/issues/193) it was also referenced to as Private Decryption Key.

**Review by mbroz2 — APPROVED** [review]

**Pull request merged**